rollup.config.mjs

```diff
@@ -21,14 +21,13 @@ if (isProd) {
   );
 }
 
-function bundleConfig({ name, entry, format = 'es' }) {
```

Comment:
All bundles generated by this config are ES modules now, since the boot bundle is generated by a separate config.